tests/test_store.py

```diff
@@ -12,10 +12,11 @@
 from schematic.models.metadata import MetadataModel
 from schematic.store.base import BaseStorage
 from schematic.store.synapse import SynapseStorage, DatasetFileView, ManifestDownload
-from schematic.utils.cli_utils import get_from_config
 from schematic.schemas.generator import SchemaGenerator
 from synapseclient.core.exceptions import SynapseHTTPError
 
+from schematic.configuration.configuration import Configuration
```

Comment:
can you move this up with the other schematic imports?

Reply:
Yep